### PR TITLE
node version upgrade - v16.14.2

### PR DESCRIPTION
**Pull request by Sreejith-Websphere**
#### Were the changes tested on
- [ ] Firefox (Desktop)
- [ ] Safari (Desktop)
- [ ] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [ ] https://validator.w3.org/checklink
- [ ] https://validator.w3.org
- [ ] IBM Equal Access Accessibility Checker (https://www.ibm.com/able/toolkit/verify/automated)
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

**Review by aknguyen7 — COMMENTED** [review]
lgtm, have you test this in demo site?

**Comment by aknguyen7** [comment]
I looked at the logs on the `demo` site, it showing that it is using the old node version, you might want to look at the `node_install.sh` to see if we can do something there

image

**Comment by aknguyen7** [comment]
I looked at the logs on the `demo` site, it showing that it is using the old node version, you might want to look at the `node_install.sh` to see if we can do something there

image

**Comment by Sreejith-Websphere** [comment]
tested in demo, please refer this link https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350/4f72040f-3d76-44b9-8c01-eabdc2dfbfcf/809bc23a-b330-4885-aec2-2fcc17cc75cc?env_id=ibm:yp:us-south
Screenshot 2022-05-16 at 5 16 08 PM

**Comment by Sreejith-Websphere** [comment]
tested in demo, please refer this link https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350/4f72040f-3d76-44b9-8c01-eabdc2dfbfcf/809bc23a-b330-4885-aec2-2fcc17cc75cc?env_id=ibm:yp:us-south
Screenshot 2022-05-16 at 5 16 08 PM

**Comment by Sreejith-Websphere** [comment]
Screenshot 2022-05-16 at 5 16 08 PM
tested in demo1, please refer this link https://cloud.ibm.com/devops/pipelines/61002d76-1c13-4502-a109-3b794b3f2350/4f72040f-3d76-44b9-8c01-eabdc2dfbfcf/809bc23a-b330-4885-aec2-2fcc17cc75cc?env_id=ibm:yp:us-south

File: scripts/build/docs_part_1.sh [hunk 1]
```diff
@@ -1,11 +1,26 @@
+#comment this below line in local if nvm already installed
```
Comment (aknguyen7) [inline scripts/build/docs_part_1.sh:1]: this part of code from line 1-14, can they be in node_install.sh? or they won't work if they are in node_install.sh?

Comment (Sreejith-Websphere) [reply scripts/build/docs_part_1.sh:1]: if we put in different script its not inheriting

Comment (aknguyen7) [reply scripts/build/docs_part_1.sh:1]: could you also check in your local build with this upgrade node, the file under src/main/content/antora_ui/build/ui-bundle.zip is generated when you build with the new timestamp? thanks

**Comment by Sreejith-Websphere** [comment]
Screenshot 2022-05-18 at 3 03 49 PM

**Comment by Sreejith-Websphere** [comment]
Screenshot 2022-05-18 at 3 03 49 PM

File: scripts/build/docs_part_1.sh (hunk 1 above)
Comment (Sreejith-Websphere) [reply scripts/build/docs_part_1.sh:1]: yes its ui-bundle.zip is generating

**Review by aknguyen7 — APPROVED** [review]

**Pull request merged**